README.md

```diff
@@ -1,3 +1,4 @@
 # ah-frontend-space
 [![Maintainability](https://api.codeclimate.com/v1/badges/9b2f555288730e8a26b4/maintainability)](https://codeclimate.com/github/andela/ah-frontend-space/maintainability)
-[![Test Coverage](https://api.codeclimate.com/v1/badges/9b2f555288730e8a26b4/test_coverage)](https://codeclimate.com/github/andela/ah-frontend-space/test_coverage)
\ No newline at end of file
+[![Test Coverage](https://api.codeclimate.com/v1/badges/9b2f555288730e8a26b4/test_coverage)](https://codeclimate.com/github/andela/ah-frontend-space/test_coverage)
+[![Build Status](https://travis-ci.org/andela/ah-frontend-space.svg?branch=ch-update-travisCI-166880069)](https://travis-ci.org/andela/ah-frontend-space)
```

Comment:
Nice work @frankopkusianwar , please ensure that the `Travis CI` badge points to the `develop` branch, not the `ch-update-travisCI-166880069` branch.

Reply:
@davi this has been resolved